src/gui/src/dbDescriptors.cpp

```diff
@@ -1313,7 +1315,10 @@ Descriptor::Actions DbNetDescriptor::getActions(std::any object) const
   }
   if(!net->getGuides().empty())
     actions.push_back(Descriptor::Action{"Route Guides", [this, gui, net]() {
-      gui->addRouteGuides(net);
+      if(guide_nets_.count(net) == 0)
+        gui->addRouteGuides(net);
+      else
+        gui->removeRouteGuides(net);
```

Comment:
Take a look at the above code for the Focus and De-focus. I think it might be better to have an action added for Route Guide and one for Hide the route guide, so it's clear what the action does.

Reply:
I agree with you but I think this can be an improvement on gui that can be done in another PR. I think for now it serves the purpose of debugging the route guides on the gui.

Reply:
I'd be okay with it being done in another PR.